server/src/client.js

```diff
@@ -101,72 +101,16 @@ class Client {
       return this.close({ error: 'Invalid handshake.', error_code: 0 });
     }
 
-    const success = (user_feed, token) => {
```

Comment:
The `user_feed` is used for permissions so that if a user's groups change in the database, we make sure that this session only uses rules from the new set of groups.  In addition, this updates the `Ruleset`s for all active requests on this session so they will error if they are no longer allowed.